### DISCUSSION TITLE
Get kube-bench and kube-hunter reports in html format?

### DISCUSSION THREAD

**Discussion by PatrickSplice** [aquasecurity/starboard#343] [Questions & Help] [answered]
I see there is a way to get the trivy and polaris reports in an html format, but I cannot find a way to do the same for the kube-hunter or kube-bench reports.  Is this functionality not there yet?

**Comment by danielpacak** [comment, marked as answer]
Currently we do not support HTML reports for kube-bench and kube-hunter.

**Comment by danielpacak** [comment]
We have  issue #396 to implement export of KubeBench reports to HTML

**Comment by krol3** [comment]
@PatrickSplice please check this link for HTML reports: https://aquasecurity.github.io/starboard/v0.12.0/cli/getting-started/#generating-html-reports
